pkgs/phps.nix

```diff
@@ -16,6 +16,21 @@ let
       pcre2 = if (prev.lib.versionAtLeast args.version "7.3") then prev.pcre2 else prev.pcre;
 
       phpAttrsOverrides = attrs: {
+        patches =
+          let
+            upstreamPatches =
+              attrs.patches or [];
+
+            ourPatches =
+              prev.lib.optionals (prev.lib.versions.majorMinor args.version == "7.2") [
+                # Patch from: https://bugs.php.net/bug.php?id=76826
+                (prev.pkgs.fetchpatch {
+                  url = "https://bugs.php.net/patch-display.php?bug_id=76826&patch=bug76826.poc.0.patch&revision=1538723399&download=1";
+                  sha256 = "sha256-aW+MW9Kb8N/yBO7MdqZMZzgMSF7b+IMLulJKgKPWrUA=";
+                })
+              ];
+          in
+          ourPatches ++ upstreamPatches;
         configureFlags =
```

Comment:
Could you add an empty line before this one?

Reply:
Done.